src/main/java/frc/robot/Robot.java

```diff
@@ -29,13 +35,17 @@ public void robotInit() {
     // and put our
     // autonomous chooser on the dashboard.
     m_robotContainer = new RobotContainer();
+    Pathfinding.setPathfinder(new LocalADStar());
```

Comment:
auto path-finding based on navgrid.json (this is not necessary at the moment but will be used in future branches---will def be used)